apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.IntegrationTests/DatastoreTransactionTest.cs

```diff
@@ -152,27 +153,32 @@ private async Task CommitTest(Func<DatastoreTransaction, Task<CommitResponse>> c
         }
 
         [Fact]
-        public void Transaction_WithCount()
+        public void Transaction_WithAggregation()
         {
             var db = _fixture.CreateDatastoreDb();
-            var keyFactory = db.CreateKeyFactory("CountTestTransaction");
+            var keyFactory = db.CreateKeyFactory("AggTestTransaction");
             var entities = new[]
             {
-                new Entity { Key = keyFactory.CreateKey("x"), ["description"] = "description for x" },
-                new Entity { Key = keyFactory.CreateKey("y"), ["description"] = "description for y" }
+                new Entity { Key = keyFactory.CreateKey("11"), ["age"] = 12, ["height"] = 5  },
+                new Entity { Key = keyFactory.CreateKey("21"), ["age"] = 12, ["height"] = 4.6  },
+                new Entity { Key = keyFactory.CreateKey("31"), ["age"] = 14, ["height"] = 4  },
+                new Entity { Key = keyFactory.CreateKey("41"), ["age"] = 11, ["height"] = 5.2  }
             };
             db.Insert(entities);
             using var transaction = db.BeginTransaction();
-            var gqlQuery = new GqlQuery { QueryString = "SELECT count(*)  as `count` FROM CountTestTransaction " };
-            var query = new Query("CountTestTransaction");
+            var gqlQuery = new GqlQuery { QueryString = "SELECT count(*)  as `count` FROM AggTestTransaction " };
```

Comment:
We probably want the gqlQuery to have sum and avg as well.

And I would prefere these tests as separate ones, one for GQL and one for AggregationQuery (even for before when the only aggregation was count).

Reply:
Done